src/profile/style.ts

```diff
@@ -26,8 +26,20 @@ export const TopContent = styled.View`
   padding: 40px 16px;
 `;
 
+const BOUNCE_SCROLLVIEW_FIX = 1000;
```

Comment:
misschien commentje hier bij want ben nie mee

Reply:
Will do (in nieuwe PR)